pkg/worker/model_status_update_job.go

```diff
@@ -43,7 +43,7 @@ func (w *Worker) ModelStatusUpdateJob(ctx context.Context, args ...interface{})
 	ctx = loaders.CTXWithLoaders(ctx, dataLoaders)
```

Comment:
Not a direct need, but you might be able to use your implementation to remove this old loader implementation since you are wrapping it generically yeah?